robot/src/test/java/us/ilite/robot/driverinput/DriverInputTest.java

```diff
@@ -0,0 +1,141 @@
+package us.ilite.robot.driverinput;
+
+import com.flybotix.hfr.util.log.ELevel;
+import com.flybotix.hfr.util.log.Logger;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.*;
+import org.mockito.junit.MockitoJUnitRunner;
+import us.ilite.common.config.SystemSettings;
+import us.ilite.common.types.input.ELogitech310;
+import us.ilite.lib.drivers.Clock;
+import us.ilite.robot.Data;
+import us.ilite.robot.commands.Delay;
+import us.ilite.robot.modules.Drive;
+import us.ilite.robot.modules.ModuleList;
+import us.ilite.robot.modules.Superstructure;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DriverInputTest {
+
+    // We're only testing integration between Superstructure and DriverInput, so we can mock this
+    @Mock private Drive mDrive;
```

Comment:
One problem with the mock annotations is that once the object is created, it's never re-created on subsequent tests. You'll want to have an "@after" annotation to null out the member,.